### PR TITLE
readme tweaks

### PR DESCRIPTION
**Pull request by sindresorhus**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ in **package.json**:
...
-	"extends": "eslint-config-vdemedes"
+	"extends": "vdemedes"
 }
 ```
 
 
 ### License
 
-MIT @ [Vadym Demedes](https://github.com/vdemedes).
+MIT @ [Vadim Demedes](https://github.com/vdemedes).
```
Comment (sindresorhus) [inline Readme.md]: Did you really misspell your name? :p

Comment (vdemedes) [inline Readme.md]: Hahahaha, no, that's how it's written in passport. Thought that is important for a license claim, right?

Comment (sindresorhus) [inline Readme.md]: But you seem to use Vadim in the other readme copyright statements.

Comment (vdemedes) [inline Readme.md]: Yeah, it's mixed up, need to fix this.

**Comment by vdemedes** [comment]
Thanks!

**Pull request merged**